lib/db_constants.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
Could we rename this file just `constants.js`? I imagine we have others that should move here as well

Reply:
Yes, we can rename this file.